src/jni_core/src/tracing_slf4j.rs

```diff
@@ -40,13 +41,22 @@ pub(crate) extern "system" fn Java_com_risingwave_java_binding_Binding_tracingSl
         let message = env.get_string(&message)?;
         let message: Cow<'_, str> = (&message).into();
 
+        let stack_trace = if stack_trace.is_null() {
+            None
+        } else {
+            Some(env.get_string(&stack_trace)?)
+        };
+        let stack_trace: Option<Cow<'_, str>> = stack_trace.as_ref().map(|c| c.into());
+        let stack_trace: Option<&str> = stack_trace.as_deref();
+
         macro_rules! event {
             ($lvl:expr) => {
                 tracing::event!(
                     target: TARGET,
                     $lvl,
                     thread = &*thread_name,
                     class = &*class_name,
+                    error = stack_trace,
```

Comment:
Should it be `stack_trace` rather than `error`?

Reply:
The `stack_trace` will also include the error message and exception class name. Just to follow the convention for recording Rust errors:

<img width="583" alt="image" src="https://github.com/user-attachments/assets/d6588e66-355d-4a32-ad4e-ded6546e9e6c" />